packages/terra-form-select/src/combobox/Frame.jsx

```diff
@@ -618,6 +625,7 @@ class Frame extends React.Component {
       value,
       ...customProps
     } = this.props;
+    delete customProps.inputId;
```

Comment:
In general I think `delete` should be used as a last resort. What issues were encountered with deconstructing the `inputId`? I see that delete was introduced in [this commit](https://github.com/cerner/terra-core/pull/2691/commits/48d7bdac314670d0ec95978991e53ebd453a1ebc#diff-b3ca3cbbd16d91fb22a942728a8132aaL607-L629).

```jsx
const {
  inputId,
  ...customProps
}
```

Reply:
`delete` is used to ensure that the `id` is rendered on the input element only and not on the select div

Reply:
Changed in aa361eb